src/pages/home-page-slides/slides-order/slides-order.js

```diff
@@ -14,10 +14,14 @@ import {
   setSlides,
   updateSlidesOrder
 } from '../../../redux/home-page-slides/home-page-slides.actions';
+import { useCommonStyles } from '../../common';
 
 const SlidesOrder = (props) => {
   const styles = useStyles();
+  const common = useCommonStyles();
```

Comment:
Can you rename this `common` variable to styles or classes?

Reply:
fixed